packages/calcite-components/src/components/panel/panel.stories.ts

```diff
@@ -440,3 +440,17 @@ export const withNoHeaderBorderBlockEnd_TestOnly = (): string =>
   html`<calcite-panel style="--calcite-panel-header-border-block-end:none;" height-scale="s" heading="My Panel"
     >Slotted content!</calcite-panel
   >`;
+
+export const contentBottomSlot = (): string => html`
```

Comment:
Might be good to merge content slots in a single story after this or the `content-top` slot PR lands.